src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -69,17 +77,20 @@ class Migrator
     /**
      * Create a new migrator instance.
      *
+     * @param  \Illuminate\Contracts\Container\Container  $container
```

Comment:
Add this as the last parameter and make it allowed to be null.

Reply:
Done